### PR TITLE
Update Appendix A - Class ID Format.md

### PR DESCRIPTION
**Pull request by cristian-recoseanu**
- Change nonstandard to non-standard
- add sentence about vendor specific functionality

### CODE REVIEW

File: docs/Appendix A - Class ID Format.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ Each class id is identified by using the [NcClassId](https://specs.amwa.tv/ms-05
 
 The lineage key of a class is set of class indices that identifies the entire lineage of the class, beginning from [NcObject](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncobject), extending down through all ancestor classes, and ending at the class in question. The key contains as many class indices as needed to describe the inheritance hierarchy.
 
-To support compatible addition of nonstandard classes to the standard control model, lineage keys allow _**authority keys**_ to be interposed in the sequence of class indices. This feature is described in [Nonstandard classIds](#nonstandard-classids).
+To support compatible addition of non-standard classes to the standard control model, lineage keys allow _**authority keys**_ to be interposed in the sequence of class indices. This feature is described in [Non-standard classIds](#non-standard-classids).
 
 A control class defined by the [Control framework](https://specs.amwa.tv/ms-05-02/) is called a _**standard class**_. The lineage key of a standard class contains no authority keys.
 
+Devices exposing vendor specific functionality would make use of non-standard classes in their device models.
```
Comment (jonathan-r-thorpe) [inline docs/Appendix A - Class ID Format.md]: The term "expose" here is not used in the rest of the document. Perhaps "modelling" instead? Suggested rewording:
```suggestion
Devices use non-standard classes to model vendor specific functionality in their device models.
```

File: docs/Appendix A - Class ID Format.md [hunk 2]
```diff
@@ -26,21 +28,21 @@ For example, for a standard class `NcXXX` whose lineage key is **1•2•12•7*
 
 • **1•2•12•7** designates class `NcXXX` , a child of the class whose parent is **1•2•12**.
 
-## Nonstandard classIds
+## Non-standard classIds
 
-A class not defined by NCA is called a _**nonstandard class**_, and its classId is called a _**nonstandard classId**_. A nonstandard classId contains at least one authority key.
+A class not defined by NCA is called a _**nonstandard class**_, and its classId is called a _**non-standard classId**_. A non-standard classId contains at least one authority key.
```
Comment (jonathan-r-thorpe) [inline docs/Appendix A - Class ID Format.md]:
```suggestion
A class not defined by NCA is called a _**non-standard class**_, and its classId is called a _**non-standard classId**_. A non-standard classId contains at least one authority key.
```

**Review by jonathan-r-thorpe — APPROVED** [review]
LGTM

**Comment by cristian-recoseanu** [comment]
Reviewed on the call on the 13th of June 2023.

**Review by jonathan-r-thorpe — APPROVED** [review]
LGTM again

**Pull request merged**